dev-tools/build-packages/deb/debian/compat

```diff
@@ -0,0 +1 @@
+8
```

Comment:
Getting this output
```
dh: Compatibility levels before 9 are deprecated (level 8 in use)
```

Reply:
Level 9 works.

Reply:
Level 10 works.

Reply:
Changed to `debhelper-compat (= 12)` in `control` file.

`compat` file delete as it was no longer needed.

Source:
- [Guide for Debian Maintainers - Section 5.19.1](https://www.debian.org/doc/manuals/debmake-doc/debmake-doc.en.pdf)
- [OpenSearch build - 2.8.0](https://github.com/opensearch-project/opensearch-build/tree/opensearch-2.8.0/scripts/pkg/build_templates/opensearch-dashboards/deb/debian) - no debian/compat file here